core/services/aws/tests/bloodDonation/donorRequestRouter.test.ts

```diff
@@ -1,45 +1,95 @@
-import { APIGatewayProxyResult } from 'aws-lambda'
-import createBloodDonationLambda from '../../bloodDonation/createBloodDonation'
+import { SQSEvent } from 'aws-lambda'
+import donorRequestRouter from '../../bloodDonation/donorRequestRouter'
 import { BloodDonationService } from '../../../../application/bloodDonationWorkflow/BloodDonationService'
-import generateApiGatewayResponse from '../../commons/lambda/ApiGateway'
-import { HTTP_CODES } from '../../../../../commons/libs/constants/GenericCodes'
-import { BloodDonationAttributes } from '../../../../application/bloodDonationWorkflow/Types'
-import { donationAttributesMock } from '../../../../application/tests/mocks/mockDonationRequestData'
+import StepFunctionOperations from '../../commons/stepFunction/StepFunctionOperations'
+import { donorRoutingAttributesMock } from '../../../../application/tests/mocks/mockDonationRequestData'
 
 jest.mock('../../../../application/bloodDonationWorkflow/BloodDonationService')
-jest.mock('../../commons/lambda/ApiGateway')
+jest.mock('../../commons/stepFunction/StepFunctionOperations')
 
-const mockBloodDonationService = BloodDonationService as jest.MockedClass<typeof BloodDonationService>
-const mockGenerateApiGatewayResponse = generateApiGatewayResponse as jest.Mock
+describe('donorRequestRouter', () => {
+  const mockEvent: SQSEvent = {
+    Records: [
+      {
+        messageId: '1',
+        receiptHandle: '1',
+        body: JSON.stringify({
+          seekerId: donorRoutingAttributesMock.seekerId,
+          requestPostId: donorRoutingAttributesMock.requestPostId
+        }),
+        attributes: {
+          ApproximateReceiveCount: '',
+          SentTimestamp: '',
+          SenderId: '',
+          ApproximateFirstReceiveTimestamp: ''
+        },
+        messageAttributes: {},
+        md5OfBody: 'md5',
+        eventSource: 'aws:sqs',
+        eventSourceARN: 'arn:aws:sqs:region:123456789012:queue',
+        awsRegion: 'region'
+      }
+    ]
+  }
 
-describe('createBloodDonationLambda', () => {
-  const { shortDescription, ...rest } = donationAttributesMock
-  const mockEvent: BloodDonationAttributes = { ...rest }
+  let mockBloodDonationService: jest.MockedClass<typeof BloodDonationService>
```

Comment:
does this really need to be let? it can be const
and 
instead of clearAllMocks(), you can use resetAllMocks()

Reply:
updated at 78987ec55bbffac5d487a6040d415ded5af9e373